packages/vscode-extension/README.md

```diff
@@ -100,4 +100,4 @@ SuiteCloud Extension for Visual Studio Code is an open source project. Pull Requ
 ## License
 Copyright (c) 2021 Oracle and/or its affiliates The Universal Permissive License (UPL), version 1.0. See [License](/LICENSE.txt) for details.
 
-By installing the Visual Studio Code Extension, you are accepting the installation of the SuiteCloud SDK dependency under this [license](/https://www.oracle.com/downloads/licenses/oracle-free-license.html).
+By installing the Visual Studio Code Extension, you are accepting the installation of the SuiteCloud SDK dependency under this [license](https://www.oracle.com/downloads/licenses/oracle-free-license.html).
```

Comment:
Hi, @antoniofruci, I wonder if we should say _By installing the SuiteCloud Extension for Visual Studio Code_, as this is the official tool name.

Reply:
Updated. Thank you